go.mod

```diff
@@ -5,6 +5,7 @@ go 1.14
 require (
 	cdr.dev/slog v1.4.1
 	cdr.dev/wsep v0.0.0-20200728013649-82316a09813f
+	github.com/blang/semver/v4 v4.0.0
```

Comment:
How does this library compare to https://github.com/Masterminds/semver? That one is used in Helm, see https://github.com/helm/helm/blob/efe2638f87b597403f34009e1029a6f0b44db8f2/go.mod#L8

Reply:
They both seem to have roughly the same number of stars and dependent libraries :raised_eyebrow: 

https://github.com/blang/semver/network/dependents?dependent_type=REPOSITORY&package_id=UGFja2FnZS0yMjY0ODE0Njcy

https://github.com/Masterminds/semver/network/dependents?package_id=UGFja2FnZS0yMjY0ODEyOTkw

I'm asking because the coder-doctor should use the same library, whichever it is. I'm not familiar with the blang one, but am open to switching if it's better!

Reply:
We use Masterminds in `/m` too: https://github.com/cdr/m/blob/master/go.mod#L18

Reply:
And we use: `https://github.com/hashicorp/go-version`

Reply:
I tried out Masterminds and it seems more feature-complete. Switching to that.